### DISCUSSION TITLE
Apks to apk merger

### DISCUSSION THREAD

**Discussion by developer-krushna** [REAndroid/APKEditor#57] [Q&A] [answered]
Hi sir.. how the apks to apk merger working in java.. can you explain the code for java

**Comment by REAndroid** [comment]
![apkmerger](https://github.com/REAndroid/APKEditor/raw/master/.github/apkmerger.png)

**Reply by developer-krushna** [reply]
I want java example not this sir

**Reply by ghost** [reply, marked as answer]
https://github.com/REAndroid/APKEditor/tree/master/src/main/java/com/reandroid/apkeditor/merge

it's better if you learn Java so you would understand everything :)
